applications/crossbar/src/api_util.erl

```diff
@@ -1266,13 +1266,19 @@ do_create_resp_envelope(Context) ->
                    [{<<"auth_token">>, cb_context:auth_token(Context)}
                    ,{<<"status">>, <<"success">>}
                    ,{<<"request_id">>, cb_context:req_id(Context)}
-                   ,{<<"revision">>, kz_term:to_binary(cb_context:resp_etag(Context))}
+                   ,{<<"node">>, kz_term:to_binary(node())}
```

Comment:
Are we worried about leaking infrastructure details here? Like, should people really know which API server processed the request when there could be a bunch sitting behind a load balancer? Kind of like we don't necessarily want to expose FreeSWITCH details when behind Kamailio?